addons/units/CfgWeapons.hpp

```diff
@@ -192,7 +192,58 @@ class CfgWeapons {
             uniformClass = QCLASS(Unit_Combat_LS_WS_WP_WB);
         };
     };
+
     // Plaid
+    class CLASS(Uniform_Combat_LS_CBPS_BP_BB): CLASS(Uniform_Base) { // CBP = Buffalo Plaid
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_LS_CBPS_BP_BB);
+        picture = QPATHOF(UI\uniform_combat_ls_cbps_bp_ca.paa);
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_LS_CBPS_BP_BB);
+        };
+    };
+    class CLASS(Uniform_Combat_LS_CDES_TP_TB): CLASS(Uniform_Base) { // CDE = Plaid Dark Earth
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_LS_CDES_TP_TB);
+        picture = QPATHOF(UI\uniform_combat_ls_cdes_tp_ca.paa);
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_LS_CDES_TP_TB);
+        };
+    };
+    class CLASS(Uniform_Combat_LS_CDFS_GP_TB): CLASS(Uniform_Base) { // CDF = Plaid Dark Foliage
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_LS_CDFS_GP_TB);
+        picture = QPATHOF(UI\uniform_combat_ls_cdfs_gp_ca.paa);
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_LS_CDFS_GP_TB);
+        };
+    };
+    class CLASS(Uniform_Combat_LS_CLES_TP_BB): CLASS(Uniform_Base) { // CLE = Plaid Light Earth
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_LS_CLES_TP_BB);
+        picture = QPATHOF(UI\uniform_combat_ls_cles_tp_ca.paa);
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_LS_CLES_TP_BB);
+        };
+    };
+    class CLASS (Uniform_Combat_LS_CLFS_GP_BB_CO): CLASS(Uniform_Base) { // CLF = Light Foliage
```

Comment:
```suggestion
    class CLASS(Uniform_Combat_LS_CLFS_GP_BB_CO): CLASS(Uniform_Base) { // CLF = Light Foliage
```

Reply:
I believe `_CO` should also be removed, and add "Plaid" to the comment, so:
```suggestion
    class CLASS(Uniform_Combat_LS_CLFS_GP_BB): CLASS(Uniform_Base) { // CLF = Plaid Light Foliage
```
?

Reply:
No, no "Plaid" in the comment ,those are all plaid. "Buffalo Plaid" is special because it's a type of Plaid.

Reply:
Well others have it, so I guess it's fine.